README.md

```diff
@@ -107,4 +107,4 @@ Stay up to date and get lakeFS support via:
 
 ## Licensing
 
-lakeFS is completely free and open source and licensed under the [Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0).
+lakeFS is completely free and open-source and licensed under the [Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0).
```

Comment:
I am not sure whether we can change this, or if this is boilerplate licensing.  @ozkatz do you know?

Reply:
I have no objection to changing this, it was phrased by us